test.js

```diff
@@ -19,3 +19,13 @@ tape('do not modify non-accented strings', function(t) {
 
 	t.end();
 });
+
+tape('.has can detect accents', function(t) {
```

Comment:
How about also adding tests for `.remove()`

Reply:
Added.